### DISCUSSION TITLE
App Image without python support?

### DISCUSSION THREAD

**Discussion by ubaldot** [vim/vim#15488] [General]
I am using the [app images](https://github.com/vim/vim-appimage/releases/tag/v9.1.0668) of both Vim and Gvim in WSL2 (Ubuntu) and I noticed that ':echo has('python3')' returns `0`. Is it wanted? Would that be possible to include python3 support in the app image?

**Comment by chrisbra** [comment]
it has dynamic python3 included, so you need to test `:echo has('python3_dynamic')`
